.storybook/preview.tsx

```diff
@@ -108,7 +116,9 @@ const MisticaThemeProvider = ({Story, context}): React.ReactElement => {
                         {(skin === TELEFONICA_SKIN || skin === TU_SKIN) && (
                             <style>{`body {font-family: "Telefonica Sans"}`}</style>
                         )}
-                        {skin === O2_SKIN && <style>{`body {font-family: "On Air"}`}</style>}
+                        {(skin === O2_SKIN || skin === O2_NEW_SKIN) && (
```

Comment:
shouldn't we use on air font for Movistar too?

Reply:
Talked via teams, we will create another PR to use on air with Movistar Skin

ticket created
https://jira.tid.es/browse/WEB-1820